tests/fuzz/zstd_helpers.c

```diff
@@ -122,6 +134,12 @@ void FUZZ_setRandomParameters(ZSTD_CCtx *cctx, size_t srcSize, FUZZ_dataProducer
     if (FUZZ_dataProducer_uint32Range(producer, 0, 1) == 0) {
       setRand(cctx, ZSTD_c_targetCBlockSize, ZSTD_TARGETCBLOCKSIZE_MIN, ZSTD_TARGETCBLOCKSIZE_MAX, producer);
     }
+
+    if (FUZZ_dataProducer_uint32Range(producer, 0, 10) == 0) {
```

Comment:
Nit: Lets enable if == 1. When the fuzzer runs out of input, it will always return 0. I'd rather that case not enable the external match finder.

```suggestion
    if (FUZZ_dataProducer_uint32Range(producer, 0, 10) == 1) {
```